api/nodes/views.py

```diff
@@ -1282,6 +1282,8 @@ class NodeFilesList(generics.ListAPIView, WaterButlerMixin, ListFilterMixin, Nod
         base_permissions.TokenHasScope,
     )
 
+    ordering = ('materialized_path',)  # default ordering
```

Comment:
After talking with @brianjgeiger, he's happy with sorting by filename for now just to get this out the door. I created https://openscience.atlassian.net/browse/OSF-5176 to cover the adding the properties and asked @lbanner to prioritize.